index.html

```diff
@@ -0,0 +1,134 @@
+                        <li class="sidebar__item">Список4</li>
+                    </ul>
+                </section>
+    
+                <section class="content section__content">
+                    <h3 class="content__title">СФУ и Россий­ский универ­ситет спорта стали парт­нёрами</h3>
+                    <section class="content__paragraph paragraph">
+                        <h4 class="paragraph__subtitle">Заголовок</h4>
+                        <p class="paragraph__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed 
+                            do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis 
+                            nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure 
+                            dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur 
+                            sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                        </p>
+                    </section>
+                    <section class="content__paragraph paragraph">
+                        <h4 class="paragraph__subtitle">Заголовок</h4>
+                        <p class="paragraph__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed 
+                            do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis 
+                            nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure 
+                            dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur 
+                            sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                        </p>
+                    </section>  
+                </section>  
+            </div>
+        </section>
+
+
+        <section class="section">
+            <h2 class="section__subtitle-center">Интересные статьи</h2>
+            <p class="section__description">Описать, о чем эти статьи, кратко</p>
+
+            <div class="cards section__cards">
+
+                <section class="card">
+                    <img class="card__image" src="D:\FrontDevelop\front-template-main\pictures\Image_1.png" alt="image">
+                    <div class="info card__info">
+                        <h4 class="info__title">Boost your conversion rate</h4> 
+                        <p class="info__text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto
+                            accusantium praesentium eius, ut atque fuga culpa,
+                            similique sequi cum eos quis dolorum.</p>
+                        <p class="info__supportingtext">Mar 16, 2020 · 6 min read</p>
+                    </div>
+                </section>
```

Comment:
section поменять на article
и посоветую это, переделать на это, 
```suggestion
                <article class="card">
                    <img class="card__image" src="D:\FrontDevelop\front-template-main\pictures\Image_1.png" alt="image">
                    <div class="card__info">
                        <h4 class="card__title">Boost your conversion rate</h4> 
                        <p class="card__text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto
                            accusantium praesentium eius, ut atque fuga culpa,
                            similique sequi cum eos quis dolorum.</p>
                        <p class="card__footer">Mar 16, 2020 · 6 min read</p>
                    </div>
                </article>
```

так в буд. будет проще, потому что это как бы один компонент, нет смысла дробить